includes/ucf-news-common.php

```diff
@@ -97,6 +99,8 @@ public static function get_story_topics( $item ) {
 			return self::get_story_terms( $item, 'post_tag' );
 		}
 	}
+	
+	add_action( 'wp_enqueue_scripts', array( 'UCF_News_Common', 'add_css' ) );
```

Comment:
We should come to a consensus on this for future development. Going to leave it here for now. The thinking was it's clear what the action hook is for since it's included in the same file as the function it's calling.